### PR TITLE
point to example for Jekyll configuration

### PR DESCRIPTION
**Pull request by afeld**

### CODE REVIEW

File: _guide/_pages/development-environments.md [hunk 1]
```diff
@@ -64,7 +66,8 @@ before_script:
 
 Now each set of scripts are much easier run in other environments: locally, in a different CI environment, etc. Moreover, think of the YAML keys as annotations for the scripts.
 
-
 **Reproducible**
 
 A good development environment should be reproducible across different computers, platforms, and environments. Reproducibility helps ensure that bugs are not idiosyncratic to any one person's bespoke computing environment--rather they are intrinsic to the repository itself such that time can be spent debugging the repository code and not the environment on a person's computer. Pinning dependencies such as language runtimes and databases to specific versions is a great way to help achieve this.
+
+[Docker]({{site.baseurl}}/project-setup/docker/) is a good way to achieve this.
```
Comment (tbaxter-18f) [inline _guide/_pages/development-environments.md:73]:
```suggestion
```

**Comment by tbaxter-18f** [comment]
I never thought of using Docker to avoid seemingly inevitable Ruby env issues. Nice addition!

File: _guide/_pages/development-environments.md (hunk 1 above)
Comment (afeld) [reply _guide/_pages/development-environments.md:73]: Can you say more about this? Use of Docker helps with both ease of setup and reproducibility.

**Comment by afeld** [comment]
Oh, yeah, it's a game changer.

File: _guide/_pages/development-environments.md (hunk 1 above)
Comment (tbaxter-18f) [reply _guide/_pages/development-environments.md:73]: Duplicates line 19?

**Comment by ryanhofdotgov** [comment]
I think I understand what @tbaxter-18f was getting at by suggesting that the 2nd link be removed, and what @afeld was trying to do by having it duplicated. 

A case can also be made that Docker also helps projects be `well-documented` too, e.g., by documenting via code what ruby version is in use, etc, so it hits all 3 principles we list on the dev env page.

I'd like to merge in this PR and but follow it up immediately with a PR that makes the Docker suggestion more prominent and tightens up the text on this page: https://github.com/18F/development-guide/pull/226

**Review by ryanhofdotgov — APPROVED** [review]

**Pull request merged**